core/src/main/res/values/strings.xml

```diff
@@ -378,6 +378,7 @@
     <string name="external_elements">External elements</string>
     <string name="interruptions">Interruptions</string>
     <string name="playback_control">Playback control</string>
+    <string name="hardware_control">Hardware control</string>
```

Comment:
How about something like "Reassign hardware buttons"? I am not happy with my suggestion (not at all) but something like that could be a bit more descriptive of what the setting actually does.